stompws.c

```diff
@@ -763,7 +784,7 @@ static const luaL_Reg lib[] = {
 };
 
 LUALIB_API int
-luaopen_stompws(lua_State *L)
+luaopen_rabid_stompws(lua_State *L)
```

Comment:
Let's keep without cujo namespace since this module will not be embedded in Rabid anymore, but distributed as an independent module.